composer.json

```diff
@@ -11,16 +11,22 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "^4.0@dev",
-        "silverstripe/cms": "^4.0@dev"
+        "silverstripe/framework": "~4.0",
+        "silverstripe/cms": "~4.0",
+        "silverstripe/asset-admin": "~1.0.0"
     },
     "require-dev": {
         "phpunit/PHPUnit": "~4.8@stable"
     },
```

Comment:
So here we should use `^4@dev` for core packages, and we need to add `silverstripe/versioned` `^1@dev` as well, and `silverstripe/admin` `^1@dev`. Do we need asset admin for something? If so, please use `^1@dev` too.

In the dev requirements can you please add `squizlabs/php_codesniffer` at `^3.0` so we can use it in Travis tests (config can be used from framework's phpcs.xml.dist).

We should also be using PHPUnit `^5.7` now. Don't worry about the stable tag.

Lastly we should add `"minimum-stability": "dev"` and `"prefer-stable": true` so we can pull dev branches in, particularly for Travis builds.

Reply:
It turns out `^4@dev` doesn't actually have any effect; `^4` is fine.